mne/viz/_brain/tests/test_brain.py

```diff
@@ -340,12 +340,11 @@ def __init__(self):
     for a, b, p, color in zip(annots, borders, alphas, colors):
         brain.add_annotation(a, b, p, color=color)
 
-    view_args = dict(view=dict(focalpoint=(1e-5, 1e-5, 1e-5)),
-                     roll=1, distance=500)
+    view_args = dict(roll=1, distance=500, focalpoint=(1e-5, 1e-5, 1e-5))
```

Comment:
This is a backward incompatible API change that will break existing working code. We can make this change (I think it's worth it) but we'll need a proper deprecation cycle